### ISSUE TITLE
there are PROBLEMS in my code.

### ISSUE THREAD

**Issue by gautsang** [microsoft/vscode-cpptools#5918]
Type: General

**Describe the bug**
- OS and Version: Windows 10
- VS Code Version: 1.47.0
- C/C++ Extension Version: 0.29.0
- Other extensions you installed (and if the issue persists after disabling them): nothing
- A clear and concise description of what the bug is.

My code always shows that there is a problem.

![image](https://user-images.githubusercontent.com/44471252/89489507-a42b3880-d7e5-11ea-9861-90f807b013d4.png)

The error means There need ")".

How can I solve this?

Thanks

**To Reproduce**
*Please include code sample and `task.json` files.*

**Expected behavior**

**Screenshots**

**Additional context**
Add any other context about the problem here including log messages from the Output window.

**Comment by sean-mcmanus** [comment]
Can you copy/paste the characters of the error so I can search for it?

Also, are you able to provide an isolated code sample that repros the error?

**Comment by gautsang** [comment]
Hi @sean-mcmanus 

the error text is below.

#define NOW (SYSTEMTIME.getSystemTime())
다음으로 확장:

(SYSTEMTIME.getSystemTime())
')'가 필요합니다.
Peek Problem (Alt+F8)
No quick fixes available


![image](https://user-images.githubusercontent.com/44471252/89601177-de0c4580-d89e-11ea-8b04-0af3239075bb.png)


I made isolated code. there is nothing error. I think the isolated code may be simple so it was not happened.

**Comment by sean-mcmanus** [comment]
Yeah, we still need more repro info -- I'm not able to repro the issue, although I can get the error if SYSTEMTIME is not a valid object. Does your code compile? Our parser is hitting some error, possibly caused by missing defines or includes being in the wrong order.

Does the error appear if you replace the macro with its definition? What do you see when you hover over SYSTEMTIME?

**Comment by gautsang** [comment]
Hi. Sorry for the late reply.

I'll give you a reference git address that causes the error. When you open this project, you will encounter countless PROBLEMS. This PROBLEMS is similar to my previous error.
If you fix the PROBLEMS of this project, my error will disappear as well.

![image](https://user-images.githubusercontent.com/44471252/91781407-33850980-ec35-11ea-96e5-0fa75c6a101a.png)

https://github.com/bitcraze/crazyflie-firmware.git
